### ISSUE TITLE
tpm2.0-tss fails to build under pyro

### ISSUE THREAD

**Issue by cardoe** [flihp/meta-measured#45]
Quick dump so I don't lose it.

```
| x86_64-poky-linux-libtool: link: x86_64-poky-linux-g++  -m64 -march=corei7 -mtune=corei7 -mfpmath=sse -msse4.2 --sysroot=/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot  -fPIC -DPIC -shared -nostdlib /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/../lib/Scrt1.o /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/../lib/crti.o /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/../lib/x86_64-poky-linux/6.3.0/crtbeginS.o  tcti/.libs/tcti_libtcti_socket_la-platformcommand.o sysapi/sysapi_util/.libs/tcti_libtcti_socket_la-changeEndian.o tcti/.libs/tcti_libtcti_socket_la-tcti_socket.o tcti/.libs/tcti_libtcti_socket_la-commonchecks.o common/.libs/tcti_libtcti_socket_la-sockets.o common/.libs/tcti_libtcti_socket_la-debug.o   -L/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/../../lib/x86_64-poky-linux/gcc/x86_64-poky-linux/6.3.0 -L/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/../../lib/x86_64-poky-linux/gcc -L/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/lib/../lib -L/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/../lib/x86_64-poky-linux/6.3.0 -L/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/x86_64-poky-linux/6.3.0 -L/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/../lib -L/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/lib -L/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib -lstdc++ -lm -lssp_nonshared -lc -lgcc_s -lgcc /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/../lib/x86_64-poky-linux/6.3.0/crtendS.o /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/../lib/crtn.o  -m64 -march=corei7 -mtune=corei7 -mfpmath=sse -msse4.2 --sysroot=/var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot -O2 -g -fstack-protector-strong -Wl,--no-undefined -Wl,--version-script=../TPM2.0-TSS/tcti/tcti_socket.map -Wl,-O1 -Wl,--hash-style=gnu -Wl,--as-needed -fstack-protector-strong -Wl,-z -Wl,relro -Wl,-z -Wl,now   -Wl,-soname -Wl,libtcti-socket.so.0 -o tcti/.libs/libtcti-socket.so.0.0.0
| /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/../lib/Scrt1.o: In function `_start':
| /usr/src/debug/glibc/2.25-r0/git/csu/../sysdeps/x86_64/start.S:104: undefined reference to `main'
| /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/libc_nonshared.a(elf-init.oS): In function `__libc_csu_init':
| /usr/src/debug/glibc/2.25-r0/git/csu/elf-init.c:86: undefined reference to `__init_array_start'
| /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/../../libexec/x86_64-poky-linux/gcc/x86_64-poky-linux/6.3.0/ld: /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot/usr/lib/libc_nonshared.a(elf-init.oS): relocation R_X86_64_PC32 against undefined hidden symbol `__init_array_start' can not be used when making a shared object
| /var/build/build/tmp/work/corei7-64-poky-linux/tpm2.0-tss/1.1.0-r0/recipe-sysroot-native/usr/bin/x86_64-poky-linux/../../libexec/x86_64-poky-linux/gcc/x86_64-poky-linux/6.3.0/ld: final link failed: Bad value
```

**Comment by cardoe** [comment]
fwiw, master builds fine with #46

**Comment by flihp** [comment]
merged #46

**Comment by cardoe** [comment]
master of tpm2-tss that is. How would you feel about adding tpm2-tss_1.2.0-rc0 to the repo?
